clap_derive/src/derives/args.rs

```diff
@@ -1,33 +1,94 @@
-// Copyright 2018 Guillaume Pinot (@TeXitoi) <texitoi@texitoi.eu>,
-// Kevin Knapp (@kbknapp) <kbknapp@gmail.com>, and
-// Andrew Hobden (@hoverbear) <andrew@hoverbear.org>
-//
-// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
-// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
-// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
-// option. This file may not be copied, modified, or distributed
-// except according to those terms.
-//
-// This work was derived from Structopt (https://github.com/TeXitoi/structopt)
-// commit#ea76fa1b1b273e65e3b0b1046643715b49bec51f which is licensed under the
-// MIT/Apache 2.0 license.
```

Comment:
We need the license text here, right?

Reply:
The reason the license is listed as removed is I originally copied the file from `subcommand.rs` which doesn't have it.  The refactor instead copied from `from_arg_matches.rs` which did have it.

Would you also like me to add it to `subcommand.rs`?

Reply:
I am not really sure how this works, but to be on the safe side, let's add it to both files.